app/controllers/my-projects/project/recommendations/add.js

```diff
@@ -65,12 +64,24 @@ export default class MyProjectsProjectRecommendationsAddController extends Contr
 
   minDate = MINIMUM_VOTE_DATE;
 
-  // hearingsSubmitted is a helper that iterates through each disposition
-  // and checks whether disposition has publichearinglocation and dateofpublichearing
-  @computed('project')
-  get hearingsSubmittedForProject() {
-    const dispositions = this.get('project.dispositions');
-    return hearingsSubmitted(dispositions);
+  // if the first object dcpPublichearinglocation and dcpDateofpublichearing properties are filled,
+  // then hearings have been submitted for that project
+  @computed('model.dispositions.firstObject.{dcpPublichearinglocation,dcpDateofpublichearing}')
+  get hearingsSubmitted() {
+    const location = this.get('model.dispositions.firstObject.dcpPublichearinglocation');
+    const date = this.get('model.dispositions.firstObject.dcpDateofpublichearing');
+    const locationAndDate = !!location && !!date;
+    return locationAndDate;
+  }
```

Comment:
This is great, I would just say that these seem like the sorts of things that would go on the model... but the problem is that the dispositions aren't always _known_ upfront, because they're not always provided, for example, in the show-geography view vs my-projects view (hence why it's usually awkward to have computed props on model relationships)...